src/device-manager/cocoa/NLNetworkInfo.h

```diff
@@ -60,7 +60,7 @@ extern const int NLThreadChannel_NotSpecified;
 @property (nonatomic, strong) NSData * ThreadPSKc; // The Thread pre-shared key commissioner, or NULL if not specified.
 
 
-- (instancetype)init NS_DESIGNATED_INITIALIZER;
+- (instancetype)init;
```

Comment:
Reviewers note: the `NS_DESIGNATED_INITIALIZER` is specified in a protected header